nasl/nasl_ssh.c

```diff
@@ -263,6 +264,18 @@ nasl_ssh_connect (lex_ctxt *lexic)
       return NULL;
     }
 
+  timeout = get_int_var_by_name (lexic, "timeout", 0);
```

Comment:
Documentation missing.

Is there also any information available which default SSH connection timeout libssh is used if no `timeout` (SSH_OPTIONS_TIMEOUT) was given?